ansible-ipi-install/roles/node-prep/templates/install-config.j2

```diff
@@ -0,0 +1,41 @@
+apiVersion: v1
+baseDomain: {{ domain }}
+metadata:
+  name: {{ cluster }}
+networking:
+  machineCIDR: {{ extcidrnet }}
+compute:
+- name: worker
+  replicas: {{ numworkers }}
+controlPlane:
+  name: master
+  replicas: {{ nummasters }}
+  platform:
+    baremetal: {}
+platform:
+  baremetal:
+    apiVIP: {{ apivip }}
+    ingressVIP: {{ ingressvip }}
+    dnsVIP: {{ ns1vip }}
+    hosts:
+{% for host in groups['masters'] %}
+      - name: {{ hostvars[host]['name'] }}
+        role: {{ hostvars[host]['role'] }}
+        bmc:
+          address: ipmi://{{ hostvars[host]['ipmi_address'] }}
```

Comment:
it's possible the address to reach would use something else than ipmi?

Reply:
I don't know --  @sreichar @schmaustech @dritch would you know if something other than ipmi would be used here?

Reply:
Keeping this as a Feature Enhancement https://github.com/openshift-kni/baremetal-deploy/issues/133